src/cmd/singularity/cli/keys_search.go

```diff
@@ -18,7 +18,7 @@ import (
 
 func init() {
 	KeysSearchCmd.Flags().SetInterspersed(false)
-	KeysSearchCmd.Flags().StringVarP(&url, "url", "u", "", "overwrite the default remote url")
+	KeysSearchCmd.Flags().StringVarP(&keyServerURL, "url", "u", "", "overwrite the default remote url")
```

Comment:
Should the `search` also read key server default from config file like `sign` and `verify`?

Reply:
This is removed completely see my comment above.